src/datetime.zig

```diff
@@ -1596,3 +1661,42 @@ test "readme-example" {
     // The time is now: Fri, 20 Dec 2019 22:03:02 UTC
 
 }
+
+test "datetime-format-ISO8601" {
+    const allocator = std.testing.allocator;
+
+    var dt = try Datetime.create(2023, 6, 10, 9, 12, 52, 49612000, null);
+    var dt_str = try dt.formatISO8601(allocator, false);
+    try testing.expectEqualStrings("2023-06-10T09:12:52+00:00", dt_str);
```

Comment:
I noticed that if no timezone is specified, UTC is used as the default timezone.